lib/Zonemaster/Backend/DB/SQLite.pm

```diff
@@ -187,20 +187,15 @@ sub create_new_test {
 
     # Search for recent test result with the test same parameters, where "$minutes"
     # gives the time limit for how old test result that is accepted.
-    my ( $recent_id, $recent_hash_id ) = $dbh->selectrow_array(
-        "SELECT id, hash_id FROM test_results WHERE params_deterministic_hash = ? AND test_start_time > DATETIME('now', '-$minutes minutes')",
+    my ( $recent_hash_id ) = $dbh->selectrow_array(
+        "SELECT hash_id FROM test_results WHERE params_deterministic_hash = ? AND test_start_time > DATETIME('now', '-$minutes minutes')",
```

Comment:
Have you considered updating this to use ? instead of string interpolation?

Reply:
Not really because I wanted to limit the change to what is needed. Would that be a considerable improvement? If so, I could do that change.

Reply:
In the updated commits the requested change is already implemented.

Reply:
Did you accidentally revert this when resolving the conflicts?

Reply:
I probably never replaced "$minutes" with "?". That can be fixed later.